src/components/AssetsDropZone/displayMessages.jsx

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  assetsDropZoneBrowseLabel: {
+    id: 'assetsDropZoneBrowseLabel',
+    defaultMessage: 'Browse your computer',
+    description: 'Upload label for the "Browse" button',
+  },
+  assetsDropZoneHeader: {
+    id: 'assetsDropZoneHeader',
+    defaultMessage: 'Drag and Drop',
+    description: 'Upload with dropzone section heading',
+  },
+  assetsDropZoneMaxFileSizeLabel: {
+    id: 'assetsDropZoneMaxFileSizeLabel',
+    defaultMessage: 'Maximum file size: {maxFileSizeMB} MB',
```

Comment:
Do we want MB to be translated? Are file sizes different across languages? Will it be translated correctly?

Reply:
I think this will be okay, since the wiki page mentions it being international multiple times in the lead paragraph: https://en.wikipedia.org/wiki/Megabyte

I'll follow up with Bill though at some point, that is a good question